lib/screens/v2/departure.ex

```diff
@@ -186,6 +186,8 @@ defmodule Screens.V2.Departure do
     stop_id
   end
 
+  def stop_id(_), do: nil
```

Comment:
When do we encounter this case? I don't think it's possible in practice to have a `Departure` where `prediction` and `schedule` are both nil, so that leaves the `stop` field of the prediction/schedule being `nil` (or some other value) as a possible match failure — but going by the typespec for both of these modules, it should only ever be a `Stop`.

Reply:
the stop is omitted in many tests

Reply:
Absent fixing the tests themselves (which I think would be best in the long term, but maybe that's a lot of extra churn to add to this one small fix, given we don't currently use factories for test data in many places), maybe we could explain why this is here? Like:

```elixir
# Expected to never occur with real data, but accommodates tests that don't fill in a stop field
# for predictions/schedules.
def stop_id(_), do: nil
```

Reply:
i've added `Stop` structs in the tests that were missing them